Tasks/PerformActionOnMachineGroup/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -2,11 +2,12 @@
 // GENERATED FILE - DO NOT EDIT DIRECTLY
 // *******************************************************
 {
-  "loc.friendlyName": "Perform action on machine group",
-  "loc.helpMarkDown": "Perform given action on machine group",
-  "loc.description": "Perform given action on machine group",
+  "loc.friendlyName": "Machine Group Actions",
+  "loc.description": "Machine Group Actions",
   "loc.input.label.ConnectedServiceName": "Subscription",
   "loc.input.label.MachineGroupName": "Machine Group",
   "loc.input.label.Action": "Action",
-  "loc.input.label.Filters": "Apply to Machines"
+  "loc.input.label.Filters": "Apply to Machines",
```

Comment:
In apply to machines markdown, we need to add - that this is noop for block and unblock.